client-side/src/stories/genericInput.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+import '../style/genericInput.scss';
+
+const GenericInput = ({ label, type='text', value, onChange, size='medium', width='20%',
```

Comment:
all props that you didn't defined as isRequired should have a default value here